contracts/LockedTokens.cdc

```diff
@@ -224,6 +224,18 @@ pub contract LockedTokens {
 
             emit LockedAccountRegisteredAsDelegator(address: self.owner!.address, nodeID: nodeID)
         }
+
+        pub fun removeNode(): @FlowIDTableStaking.NodeStaker {
+            let node <- self.nodeStaker <- nil
+
+            return <-node!
```

Comment:
Maybe just make this function return an optional, and not force-unwrap here? 
Then in case the field is already nil, the function is just a "no-op" instead of aborting.